c7n/commands.py

```diff
@@ -316,63 +315,6 @@ def logs(options, policies):
     sys.exit(1)
 
 
-def _schema_get_docstring(starting_class):
-    """ Given a class, return its docstring.
-
-    If no docstring is present for the class, search base classes in MRO for a
-    docstring.
-    """
-    for cls in inspect.getmro(starting_class):
-        if inspect.getdoc(cls):
-            return inspect.getdoc(cls)
-
-
-def schema_completer(prefix):
-    """ For tab-completion via argcomplete, return completion options.
-
-    For the given prefix so far, return the possible options.  Note that
-    filtering via startswith happens after this list is returned.
-    """
-    from c7n import schema
-    load_available()
-    components = prefix.split('.')
-
-    if components[0] in provider.clouds.keys():
```

Comment:
is components[0] always aws?

Reply:
its either the provider, `mode`, or naked in which case we prefix aws .


ie.  these are all valid

```
custodian schema aws
custodian schema ec2
custodian schema 
custodian schema gcp.instance
custodian schema gcp.instance.filters
```